src/adapter/src/coord/appends.rs

```diff
@@ -209,7 +209,7 @@ impl Coordinator {
     /// All applicable pending writes will be combined into a single Append command and sent to
     /// STORAGE as a single batch. All applicable writes will happen at the same timestamp and all
     /// involved tables will be advanced to some timestamp larger than the timestamp of the write.
-    #[tracing::instrument(level = "debug", skip_all)]
+    #[instrument(name = "coord::group_commit_initiate", skip_all)]
```

Comment:
This happens every second due to timeline advancements, I don't know if that's considered too busy or not for info, just an FYI.

Reply:
Yes, I think this is ok. One of the things Dan and I agreed on is that persist CAS operations (and presumably blob::set) must have some more explicit parent span. CAS and blob::set trace at info, so anything that calls them should also be info or above.